Gemfile.lock

```diff
@@ -76,7 +76,7 @@ GEM
       activerecord (>= 3.2, < 6.0)
       rake (~> 10.4)
     arel (3.0.3)
-    bootstrap-sass (2.3.1.2)
+    bootstrap-sass (2.3.1.3)
```

Comment:
Probably because I didn't update the version matcher in the Gemfile correctly \*sigh\*
Cheers!